common/archiver/filestore/visibilityArchiver.go

```diff
@@ -197,7 +198,8 @@ func (v *visibilityArchiver) query(
 
 	response := &archiver.QueryVisibilityResponse{}
 	for idx, file := range files {
-		encodedRecord, err := readFile(path.Join(dirPath, file))
+		// #nosec
+		encodedRecord, err := ioutil.ReadFile(path.Join(dirPath, file))
```

Comment:
@yycptt  please review and provide comment.

Reply:
Wasn't 100% sure on this one. How does the visibilityArchiver work? Is there a danger a user outside the system could supply input

Reply:
Here the archiver will try to read each file and parse the content into a visibility record. If user outside the system provides some random input, the parsing will fail. If the input is in correct format, the parsed result will be returned (if the record matches some filter) to user. Cadence system won't execute any other logic based on the file content. I don't think it will be a danger if user outside the system can supply input. Also when starting the system, there's a configuration for the permission of the folder. Unauthorized user doesn't have permission to change file contents.